Simbody/Visualizer/simbody-visualizer/simbody-visualizer.cpp

```diff
@@ -2043,8 +2043,12 @@ static void readDataFromPipe(int srcPipe, unsigned char* buffer, int bytes) {
     int totalRead = 0;
     while (totalRead < bytes) {
         auto retval = READ(srcPipe, buffer + totalRead, bytes - totalRead);
+        int attempts = 0;
+        while (retval == -1 && errno == 4 && ++attempts < 5) {
```

Comment:
Would it be better to use the error name here? I think errno == 4 is `EINTR`. I wonder if `errno == EINTR` would work.

https://man7.org/linux/man-pages/man3/errno.3.html:

> The error numbers that correspond to each symbolic name vary across
       UNIX systems, and even across different architectures on Linux.

https://docs.microsoft.com/en-us/cpp/c-runtime-library/errno-doserrno-sys-errlist-and-sys-nerr?view=vs-2019

Reply:
agree, it works, pushed this change